tests/unix_stream.rs

```diff
@@ -77,6 +82,7 @@ fn unix_stream_connect() {
     handle.join().unwrap();
 }
 
+#[cfg(unix)]
```

Comment:
The code has `cfg(windows)`, but here you're making it Unix only?

Reply:
Currently I was not exposing the internal blocking UDS types for windows, so there is no `from_std` function on `UnixStream` in Windows, and this test wouldn't really make sense on windows